addons/website/views/website_views.xml

```diff
@@ -39,8 +39,11 @@
                         </div>
                         <div name="other">
                             <group name="other">
+                                <field name="language_count" invisible="1"/>
                                 <field name="company_id" options="{'no_open': True, 'no_create': True}" groups="base.group_multi_company"/>
-                                <field name="default_lang_id" options="{'no_open': True, 'no_create': True}" groups="base.group_no_one"/>
+                                <field name="language_ids" widget="many2many_tags" options="{'no_open': True, 'no_create': True}"/>
```

Comment:
Missing `required`, one can simply remove the one set by default and save, the website won't have a lang and it will crash when trying to access it